styles/variables-gallium.less

```diff
@@ -43,22 +44,13 @@
 @agate-heading-title-decoration-opacity: 0.8;
 @agate-heading-title-decoration-transform: skew(-10deg);
 
-// Icon Sizes
-// ---------------------------------------
-@agate-icon-small-size: 28px;
-@agate-icon-huge-font-size: 400px;
```

Comment:
I think removing this drastically changes the size of the huge icon in Gallium. It would go from 400px to a little over 200px. Is this intentional?